pandas/tests/frame/methods/test_value_counts.py

```diff
@@ -100,3 +100,47 @@ def test_data_frame_value_counts_empty_normalize():
     expected = pd.Series([], dtype=np.float64)
 
     tm.assert_series_equal(result, expected)
+
+
+def test_data_frame_value_counts_dropna_true(nulls_fixture):
```

Comment:
Is it possible to parameterize this test with `dropna` instead of 2 separate tests?

Reply:
I am not sure how to do that while using the `nulls_fixture`. Any suggestion is welcome!

Reply:
You can simply add another parametrization:

```
@pytest.mark.parametrize("x", [1, 2])
def test_data_frame_value_counts_dropna_true(nulls_fixture, x):
    pass
```

But I don't know if this wouldn't make it more complicated here.

Reply:
The issue is that the two input and expected dataframes are quite different in the two tests, so they would need to be defined as part of the parametrization. But both are also defined using the values from the `nulls_fixture`, so I would somehow need to use the fixture values in the parametrization, and I'm not sure how I can do both.

Reply:
Ah now I get it. if else is the only way here, but I think this would make it more complicated than is worth